src/xdp-utils.c

```diff
@@ -1994,7 +1994,11 @@ xdg_app_info_load_brwap_info (XdpAppInfo *app_info,
   instance = xdp_app_info_get_instance (app_info);
 
   if (instance == NULL)
-    return 0;
+    {
+      g_set_error_literal (error, G_IO_ERROR, G_IO_ERROR_FAILED,
+                           "Could not find instance-id in processes /.flatpak-info");
```

Comment:
```suggestion
                           "Could not find instance-id in process's /.flatpak-info");
```

Reply:
I'm not a native speaker, but in this case, wouldn't the appropriate form be `process'` (without the extra `s`)?

Reply:
Honestly nobody agrees, so both ways are OK. I like `s's` because it is unambiguously singular, whereas `s'` can be unclear. [This page](https://oneminuteenglish.org/en/possessive-nouns-ending-in-s/) that I just found argues that `process'` is better because the word has more than one syllable. I've never even heard of that rule before. Then [this page](https://www.thesaurus.com/e/grammar/whats-the-rule-for-doing-a-possessive-after-the-word-s/) says the monosyllable rule only applies to historical and biblical names. I've *really* never heard of that rule before! So I've learned multiple new things about possessive nouns today....

When I was younger, I mostly saw people writing `s'` and that is what my teachers used in school, but they also didn't use Oxford commas, which nowadays would be a quite unfashionable omission, so I don't think I'll take style advice from them. :P Nowadays, I think I see `s's` more commonly than before, but maybe I'm totally wrong about that.